### PR TITLE
Add demo: canvas captureStream() to peer connection 

### PR DESCRIPTION
**Pull request by samdutton**
**Description**

Additional canvas `captureStream()` demo. 

Also includes updates to fix canvas-video demo display problems on smaller viewports.

**Purpose**

Show stream from canvas used as 'local stream' for peer connection.

### CODE REVIEW

File: src/content/capture/canvas-pc/js/main.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
+/*
+ *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
```
Comment (KaptenJansson) [inline src/content/capture/canvas-pc/js/main.js]: Update copyright year.

File: src/content/capture/canvas-pc/js/test.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
```
Comment (KaptenJansson) [inline src/content/capture/canvas-pc/js/test.js]: Update copyright year.

File: src/content/capture/canvas-pc/index.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+    <p>The teapot is drawn on the canvas element using WebGL. A stream is captured from the canvas using its <code>captureStream()</code> method and streamed via a peer connection to the video element on the right.</p>
+
+    <p>View the browser console to see logging.</p>
+
+    <p>Several variables are in global scope, so you can inspect them from the console: <code>canvas</code>, <code>video</code>, <code>localPeerConnection</code>, <code>remotePeerConnection</code> and <code>stream</code>.</p>
+
+    <p>For more information about RTCPeerConnection, see <a href="http://www.html5rocks.com/en/tutorials/webrtc/basics/" title="HTML5 Rocks article about WebRTC by Sam Dutton">Getting Started With WebRTC</a>.</p>
+
+    <a href="https://github.com/webrtc/samples/tree/gh-pages/src/content/capture/canvas-stream" title="View source for this page on GitHub" id="viewSource">View source on GitHub</a>
```
Comment (KaptenJansson) [inline src/content/capture/canvas-pc/index.html]: canvas-stream > canvas-pc

**Comment by KaptenJansson** [comment]
Could perhaps try to use more promises in general for the various APIs?

File: src/content/capture/canvas-pc/js/test.js (hunk 2 above)
Comment (samdutton) [inline src/content/capture/canvas-pc/js/test.js]: D'oh! Done. Might have to write a test for this :).

File: src/content/capture/canvas-pc/index.html (hunk 3 above)
Comment (samdutton) [inline src/content/capture/canvas-pc/index.html]: Ah yes — noticed this... In next PR.

**Comment by samdutton** [comment]
Agree re promises — will take a look.

**Comment by KaptenJansson** [comment]
We can fix the promises later using codemod if you prefer (if time is lacking). LGTM.

**Pull request merged**